docs/how-to/jupyter-notebook.rst

```diff
@@ -103,14 +103,14 @@ You should see an output similar to this:
 
 .. code-block:: bash
 
-        - intel-pytorch = intel/intel-extension-for-pytorch:2.1.20-xpu-idp-jupyter
-        - intel-tensorflow = intel/intel-extension-for-tensorflow:2.15.0-xpu-idp-jupyter
+        - pytorch-intel= intel/intel-extension-for-pytorch:2.1.20-xpu-idp-jupyter
+        - tensorflow-intel = intel/intel-extension-for-tensorflow:2.15.0-xpu-idp-jupyter
 
 Select one of them and create a notebook as follows:
 
 .. code-block:: bash
 
-   dss create my-itex-notebook --image=intel-tensorflow
+   dss create my-itex-notebook --image=tensorflow-intel
```

Comment:
Can you also check other sections where we create notebooks, to see whether updates are needed or not? I guess not bcs this is only related only, but double-check pls. Pages I found:
- https://documentation.ubuntu.com/data-science-stack/en/latest/tutorial/get-started/#launch-a-notebook
- Output in https://documentation.ubuntu.com/data-science-stack/en/latest/how-to/dss/#start-dss

Reply:
I checked as well and indeed there no other references to intel in our docs that need to be updated. Resolving